instaloader/__main__.py

```diff
@@ -15,6 +15,31 @@
 from .instaloadercontext import default_user_agent
 from .lateststamps import LatestStamps
 
+def targetlist_parser(targets: List[str]) -> List[str]:
+    """Take the list of targets and remove
+    - Blanklines
+    - lines starts withs #
+    - # from line, used as line comment
+
+    Args:
+        targets (List[str]): List contains args strings
+
+    Returns:
+        List[str]: Clean list
```

Comment:
This is not the right syntax for our docstring; see https://www.sphinx-doc.org/en/master/tutorial/describing-code.html or the other docstrings in Instaloader for a reference of the syntax.